CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contribution Guidelines
+
+We follow the Contributor Covenant. You can read it, and where to report abuse in our [Code of Conduct](./CODE_OF_CONDUCT.md).
+
+## How to Contribute
+
+If you'd like to fix a bug, add a feature, or just generally tinker with
+`styled-system`, please start by forking the repository, and creating a new
+branch. Then install dependencies by running `npm install` (we use `npm`, you're
+free to use `yarn`, just take care to not commit `yarn.lock`).
+
+You can start the documentation development server by executing `npm start`. By
```

Comment:
I just ripped all this out, but don't worry, I can update this docs after merging 😬